### PR TITLE
Fix Get-AzureRmVM paging with ResourceGroupName

### PR DESCRIPTION
**Pull request by jabbera**
Add paging support to Get-AzureRmVM when -ResourceGroupName parameter is set.

## Description

Get-AzureRmVM -ResourceGroupName blah is only returning 50 results even if there are more then 50 vms in an RG.

You may not believe me, but this test: https://github.com/Azure/azure-powershell/blob/preview/tools/InstallationTests/NetcoreTests/Compute/VirtualMachineTests.ps1#L1988 is currently failing because the paging code does not exist when filtering by resource group name. I have no idea why it used to work unless more the 50 results used to be returned by the endpoint for this call.

## Checklist

- [X] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [X] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [X] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [X] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
@hyonholee Hey Hyonho, would you mind reviewing this PR when you get the chance?

**Comment by jabbera** [comment]
Sorted the CI build.

**Comment by changov** [comment]
Sorry, folks, this appears to be a regression from the server side. We just enabled paging for the GET VMs in Resource Group query but it was not done properly with an api-version check. We'll correct this in the next few days. The client-side fix will probably also be needed, but deferring to Hyonho & co. to figure out exactly what & where.

File: src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Operation/GetAzureVMCommand.cs [hunk 1]
```diff
@@ -158,7 +158,21 @@ public override void ExecuteCmdlet()
...
                     var psResultListStatus = new List<PSVirtualMachineListStatus>();
-                    psResultListStatus = GetPowerstate(vmListResult, psResultListStatus);
+
+                    while (vmListResult != null)
+                    {
+                        psResultListStatus = GetPowerstate(vmListResult, psResultListStatus);
+
+                        if (!string.IsNullOrEmpty(vmListResult.Body.NextPageLink))
+                        {
+                            vmListResult = this.VirtualMachineClient.ListAllNextWithHttpMessagesAsync(vmListResult.Body.NextPageLink)
```
Comment (hyonholee) [inline src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Operation/GetAzureVMCommand.cs]: use "ListNextWithHttpMessagesAsync"

**Comment by hyonholee** [comment]
Please hold this PR.  I think we need to use the updated the client library which will use 2018-10-01 version.  (the client library for 2018-10-01 is not yet published)

**Pull request closed without merging**